compiler/executable/update/instructions.rs

```diff
@@ -0,0 +1,37 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use std::fmt;
+
+use crate::executable::insert::{ThingPosition, TypeSource};
+
+#[derive(Debug)]
```

Comment:
We not use the general `write_instructions`? They're the same, no?

Reply:
I saw your comment earlier, but if they're the same data structure then we can keep them the same IMO, just process them differently

Reply:
RIght ok - let's leave it at that!